src/understanding-workflow/concepts.md

```diff
@@ -108,11 +108,11 @@ Note that during the deploy, your application will be running in a mixed mode.
 
 Workflow treats all persistent services such as databases, caches, storage,
 messaging systems, and other [backing services][] as resources managed
-separtely from your application. This philosophy aligns with Twelve-Factor
+separately from your application. This philosophy aligns with Twelve-Factor
```

Comment:
not sure if there is one above, but if not, can we make `Twelve-Factor` link to http://12factor.net/?